eos-prune-printers.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Endless prune printers
+After=cups.service
+ConditionPathExists=!/var/eos-printers-pruned
```

Comment:
Let's make this friendlier to typical path usage and namespaced to eos4 in case we need to do this again for the eos4 -> eos5 transition.

How about `/var/lib/eos4-printers-pruned` (with the script adjusted, too)?

Reply:
okay